wandb/sdk/artifacts/artifact.py

```diff
@@ -562,9 +557,11 @@ def ttl(self, ttl: Union[timedelta, ArtifactTTL, None]) -> None:
 
     @property
     def aliases(self) -> List[str]:
-        """The aliases associated with this artifact.
+        """Labels assigned to an artifact. Aliases are mutable.
```

Comment:
```suggestion
        """List of semantically relevant references or identifying "nicknames" assigned to an artifact which can be used to programmatically reference a model version. Aliases are mutable.
```
Label is often more synonymous to a tag (more like a keyword that can be applied to many models).

Reply:
I agree!

Reply:
oh whoops that was a typo on my end, didn't mean model :). 3

Reply:
Sounds great!